GothicVRProject/GothicVR
# Bugfix/armor misalignment v2

# To test
1. Start game with Items and NPCs
2. Look if NPC heads are correct
3. Look if items, world, everything has textures
4. Check also if general features like barrier is working.
